### PR TITLE
removing text and removing link to the overview section - I think it …

### PR DESCRIPTION
**Pull request by bpfarmer**
…might be too much text for now - we can put it back later when we have more sweet graphics :)

### CODE REVIEW

**Comment by dlubarov** [comment]
Please remove `.DS_Store`s; we can add that to `.gitignore`

File: static/common.css [hunk 1]
```diff
@@ -121,6 +121,11 @@ section h2 {
   margin-bottom: 0.5em;
 }
 
+#section-pitch {
+    padding-top: 60px;
+    padding-bottom: 60px;
```
Comment (dlubarov) [inline static/common.css]: I'd prefer to keep this symmetric; do you think things are too spaced out overall? If so we could reduce the padding on `section` a bit.

File: static/index.xhtml [hunk 2]
```diff
@@ -140,8 +139,8 @@
     <section id="section-pitch" class="checkered-section">
       <div class="content">
         <div class="paragraph-size">
-          <header><h2>A scalable protocol for succinctly verifiable computation.</h2></header>
-          <p>Mir is a cryptocurrency platform that supports arbitrary computations. Rather than verifying a block by reexecuting each included computation, Mir uses succinct arguments to prove the correctness of each state transition.</p>
+          <header><h2>Trust, succinctly.</h2></header>
+          <p>Mir is a distributed ledger that supports decentralized applications. <br /><br />It uses cryptographic proofs to verify transactions, improving security and scalability.</p>
```
Comment (dlubarov) [inline static/index.xhtml]: nit: separate `<p>` would be best to make sure it's consistent with other paragraph margins

File: static/index.xhtml [hunk 3]
```diff
@@ -173,9 +172,10 @@
         </svg>
 
         <div class="text">
-          <header><h2>The problem with reexecution</h2></header>
-          <p>In other blockchain designs, nodes must reexecute each smart contract themselves in order to verify that it was carried out correctly. In a large network, each computation might be reexecuted thousands or even millions of times.</p>
-          <p>This redundancy is incredibly wasteful. In order to keep costs under control, these blockchains charge a “gas” fee for each transaction proportional to its computational cost.</p>
+          <header><h2>The Problem with Blockchains</h2></header>
```
Comment (dlubarov) [inline static/index.xhtml]: If you prefer title case (fine with me) please tweak the couple other headers (`The team` and `Stay tuned`).

File: static/common.css (hunk 1 above)
Comment (bpfarmer) [reply static/common.css]: Yeah I'd like to reduce the section height overall, it seems like there might be a bit too much whitespace?

File: static/index.xhtml (hunk 3 above)
Comment (bpfarmer) [reply static/index.xhtml]: Will do

**Pull request merged**